packages/stream_video/lib/src/models/call_participant_state.dart

```diff
@@ -73,6 +80,14 @@ class CallParticipantState
     CallReaction? reaction,
     ViewportVisibility? viewportVisibility,
   }) {
+    final levels = audioLevels;
```

Comment:
I'm not sure `copyWith` is the correct place for this logic. We can't assume that it will always be used to update the state with latest data. It might be used to override the whole state or any other reason. We should add `audioLevels` as a `copyWith` parameter and move this logic to the place where new/latest audioLevel is actually updated. Probably `sfuUpdateAudioLevelChanged`method.

Reply:
Changed it: https://github.com/GetStream/stream-video-flutter/pull/882/commits/f9847e147c07b5e9bcdc060c88dbc9b3d56e45b0